modules/assisted-installer-dpu-introduction.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/dpu-hardware-offload.adoc
+
+:_content-type: CONCEPT
+[id="assisted_installer_overview_{context}"]
+= Assisted Installer overview
+
+The assisted installer is a layer on top of the {product-title} installer that simplies the {product-title} installation for a number of different platforms, but focuses on bare metal deployments.
+
+The high level flow for the user is as follows: 
+
+. Create a cluster in the assisted installer UI
```

Comment:
@romfreiman @tsorya 
this is referring to SaaS AI which doesn't have support for ARM
should we remove this ?

Reply:
high-level flow
- Added a hyphen.